pokemonterminal/scripter.py

```diff
@@ -53,8 +24,27 @@ def change_wallpaper(image_file_path):
     if not isinstance(image_file_path, str):
         print("A image path must be passed to the change wallpapper function.")
         return
-    if sys.platform == "darwin":
-        script = osa_script_fmt.format(image_file_path)
-        __run_osascript(str.encode(script))
-    elif sys.platform == "linux":
-        os.system(__linux_create_wallpaper_script(image_file_path))
+    providers = get_current_adapters()
+    if len(providers) > 1:
+        # All this if is really not supposed to happen at all whatsoever
+        # really what kind of person has 2 simultaneous D.E???
+        print("Multiple providers found select the appropriate one:")
+        [print(str(x)) for x in providers]
+        print("If some of these make no sense or are irrelevant please file" +
+              "an issue in https://github.com/LazoCoder/Pokemon-Terminal")
+        print("=> ", end='')
+        inp = None
+        while inp is None:
+            try:
+                inp = int(input())
+                if inp >= len(providers):
+                    raise ValueError()
+            except ValueError as _:
+                print("Invalid number, try again!")
+        target = providers[inp]
+    elif len(providers) <= 0:
+        print("Your desktop environment isn't supported by this time.")
```

Comment:
Did you mean `isn't supported by this theme`?

Reply:
I think he meant that it is not supported now, but it may be in the future. As in "Your desktop environment isn't supported **at** this time." or "isn't supported yet".

Reply:
Yep @LazoCoder got it, my bad.